tests/unit/api/objectPutACL.js

```diff
@@ -151,14 +158,9 @@ describe('putObjectACL API', () => {
                 'x-amz-grant-full-control':
                     'emailaddress="sampleaccount1@sampling.com"' +
                     ',emailaddress="sampleaccount2@sampling.com"',
-                'x-amz-grant-read':
-                    `uri=${constants.logId}`,
-                'x-amz-grant-read-acp':
-                    'id="79a59df900b949e55d96a1e698fbacedfd6e09d98eac' +
-                    'f8f8d5218e7cd47ef2be"',
-                'x-amz-grant-write-acp':
-                    'id="79a59df900b949e55d96a1e698fbacedfd6e09d98eac' +
-                    'f8f8d5218e7cd47ef2bf"',
+                'x-amz-grant-read': `uri=${constants.logId}`,
+                'x-amz-grant-read-acp': `id="${ownerID}"`,
```

Comment:
why quotes around the id?  I've been fixing that in another pr.  I don't see why we did this originally.